lib/ui/platform_dispatcher.dart

```diff
@@ -149,6 +149,14 @@ class PlatformDispatcher {
     _onPlatformConfigurationChangedZone = Zone.current;
   }
 
+  /// The current list of displays.
+  ///
+  /// If any of their configurations change, [onMetricsChanged] will be called.
+  ///
```

Comment:
Maybe document some of the current platform-specific limitations? Presumably, this is just empty for desktops right now?

Reply:
I added some docs.

Reply:
The same caveat should likely also be documented on the FlutterView.display property?